apisix/utils/log-util.lua

```diff
@@ -120,7 +122,9 @@ local function get_full_log(ngx, conf)
         consumer = consumer,
         client_ip = core.request.get_remote_client_ip(ngx.ctx.api_ctx),
         start_time = ngx.req.start_time() * 1000,
-        latency = (ngx_now() - ngx.req.start_time()) * 1000
+        latency = latency,
+        upstream_latency = upstream_latency,
+        apisix_latency = apisix_latency
```

Comment:
I think use `apisix_latency` is confused, refer another discussment https://github.com/apache/apisix/issues/6029.
The `apisix_latenct` includes client sending data via network, not apisix exclusive. From my experience, it is large when tranfer big file.

Reply:
Maybe in this PR do like 
`apisix_latency = apisix_latency - upstream_latency` to get the real apisix_latency, cc @spacewander

Reply:
The latency from the client exists through the whole request process.
For file upload, it affects the apisix latency much.
But for file download, it affects the upstream latency much.
Both latencies contain the latency from the client.
So it's OK to keep the current behavior.